docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.9'
+services:
+  ruuter:
+    container_name: ruuter
+    build:
+      context: .
+    environment:
+      - logging.level.root=INFO
```

Comment:
Can the `INFO` level be default level?
If it's baked into docker image and there is a need for different level then the env value can be overloaded.

Reply:
I personally would avoid repeating and add line about env changes into readme. But the other way is also solid solution.